TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>So. Unfortunately, this won't work -- if you look at the aperture, you'll see it's all the pixels on the right-hand side. It looks like TESS just missed getting Trappist-1 in the FFI. I foolishly thought that because FFI results are returned that there was actual data there, but it's just in one of the buffer rows. </p><p> </p><p>We should still complete this exercise, but we'll need a different target. You're welcome to pick your favorite, but HD10180 has been observed multiple times and has light curve files (so it is definitely in the FFI)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>updated! please review latest version, light curve of HD10180 looks odd to me. </p><p> </p>